R/plotBeta.R

```diff
@@ -51,7 +51,7 @@ plotBeta <- function(alpha, beta) {
 #'
 #' @example examples/plotBetaDiff.R
 #'
-#' @importFrom graphics par axis polygon mtext
+#' @importFrom ggplot2 geom_line ggtitle geom_area
```

Comment:
not sure if these need to be imported here at all, because below we use `ggplot2::` prefix?